src/router/routing.rs

```diff
@@ -1,43 +1,18 @@
-use super::{DevAddrFilter, EuiFilter};
 use crate::{KeyedUri, PublicKey, Result};
-use helium_proto::{routing_information::Data as RoutingData, RoutingInformation};
 use slog::{warn, Logger};
 use std::{convert::TryFrom, sync::Arc};
 
 #[derive(Clone, Debug)]
 pub struct Routing {
```

Comment:
I think we need to get rid of Routing altogether and have routers be a DNS name(s) for regional delivery right? this would remove the public key of those routers which will be something we need to talk about.

Reply:
Routing has been removed. 
I'm unsure of the implications of removing the router public keys. But like you said, that's something we can talk about.